uc-crux-llvm/src/UCCrux/LLVM/Config.hs

```diff
@@ -181,6 +191,18 @@ ucCruxLLVMConfig = do
                    ["re-explore"]
                    "Re-explore functions that have already been explored (i.e., have logs)"
                    $ Crux.NoArg $
-                     \opts -> Right opts {reExplore = True}
+                     \opts -> Right opts {reExplore = True},
+                 Crux.Option
+                   []
+                   ["skip-functions"]
+                   "List of functions to skip during exploration"
+                   $ Crux.ReqArg "FUN" $
+                     \v opts -> Right opts {skipFunctions = v : skipFunctions opts},
+                 Crux.Option
+                   []
+                   ["unsound-overrides"]
+                   (Text.unpack unsoundOverridesDoc)
+                   $ Crux.NoArg $
+                     \opts -> Right opts {unsoundOverrides = True}
```

Comment:
Should we default to True? It seems like there is little downside (in the abstract)

Reply:
I was worried because the reporting claims things like "this function is always safe", which wouldn't always be true if said function (transitively) called these overrides, but combined with tracking the use of these overrides so that we can report precisely when they were used, I agree that there is little downside. I'll just go ahead and remove this configuration option entirely.